tests/smoke/support/api/http.ts

```diff
@@ -0,0 +1,52 @@
+import join from 'join-path'
+import fetch, { BodyInit, Response } from 'node-fetch'
+import { User } from '../types'
+import { config } from '../config'
+import _ from 'lodash'
```

Comment:
Why not use `lodash-es` here?

Reply:
we could but there it doesnt matter, does not get packed